engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/serialization/jsonpayload/ConfluentJsonPayloadDeserializerFactory.scala

```diff
@@ -15,6 +15,46 @@ import tech.allegro.schema.json2avro.converter.JsonAvroConverter
 
 import scala.reflect.ClassTag
 
+/**
+  * ConfluentJsonPayloadDeserializer converts json payload to avro.
+  * It uses avro schema which has impact on json format requirements:
+  * 1. Avro schema is used to deserialize json payload to GenericRecord or SpecificRecord.
+  *    Values should be encoded with avro conversions (see example below).
+  * 2. Allegro json2avro decoder is used instead of confluent decoder.
+  *    Decoded json should have flattened object structure (see example below).
+  *
+  * TODO: Use confluent json schema instead of avro schema for json payloads.
```

Comment:
Please reformat this notice to sth like "TODO: Add support to json payloads specified in json schema". Current solution (json payload described in avro schema is not completely egzotic - e.g. NiFi uses this concept for jsons, csv and xmls. Thanks to this approach user can have one standard for schema used for multiple input types

Reply:
Changed that notice.